packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -932,6 +932,10 @@ export default {
             this.setSubmittingState(true);
 
             if (this.isFormValid()) {
+                const now = new Date();
+                if (!this.isLoggedIn) {
+                    this.$cookies.set('je-gtm-event', 'engagement|form_checkout_guest|success', now.setMinutes(now.getMinutes() + 1));
+                }
```

Comment:
Maybe only create variable if you need it ....

```suggestion                
                if (!this.isLoggedIn) {
                    const now = new Date();
                    this.$cookies.set('je-gtm-event', 'engagement|form_checkout_guest|success', now.setMinutes(now.getMinutes() + 1));
                }
```

Reply:
Used `max-age` instead.